### PR TITLE
Add INTO THE DEEP Official Field Image and Juice Custom Alternative Images

### PR DESCRIPTION
**Pull request by kidsonfilms-python-rules**
images also shown here for ease of access:
field-2024-juice-light
field-2024-juice-dark
field-2024-official

### CODE REVIEW

**Comment by j5155** [comment]
Closes #9

File: src/main/kotlin/com/noahbres/meepmeep/MeepMeep.kt [hunk 1]
```diff
@@ -319,6 +319,18 @@ open class MeepMeep @JvmOverloads constructor(private val windowSize: Int, fps:
                 colorManager.isDarkMode = false
                 rotated(ImageIO.read(classLoader.getResourceAsStream("background/season-2023-centerstage/field-2023-juice-light.png")), Math.toRadians(90.0))
             }
+            Background.FIELD_INTOTHEDEEP_OFFICIAL -> {
+                colorManager.isDarkMode = false
+                ImageIO.read(classLoader.getResourceAsStream("background/season-2024-intothedeep/field-2024-official.png"))
+            }
+            Background.FIELD_INTOTHEDEEP_JUICE_DARK -> {
+                colorManager.isDarkMode = false
```
Comment (rbrott) [inline src/main/kotlin/com/noahbres/meepmeep/MeepMeep.kt:327]: Probably want true here

File: src/main/kotlin/com/noahbres/meepmeep/MeepMeep.kt [hunk 2]
```diff
@@ -319,6 +319,18 @@ open class MeepMeep @JvmOverloads constructor(private val windowSize: Int, fps:
                 colorManager.isDarkMode = false
                 rotated(ImageIO.read(classLoader.getResourceAsStream("background/season-2023-centerstage/field-2023-juice-light.png")), Math.toRadians(90.0))
             }
+            Background.FIELD_INTOTHEDEEP_OFFICIAL -> {
```
Comment (rbrott) [inline src/main/kotlin/com/noahbres/meepmeep/MeepMeep.kt:322]: I prefer `INTO_THE_DEEP` for easy reading and to match the service mark (as appears to have been done in the past)

**Review by rbrott — COMMENTED** [review]
Thanks for the patch! Left a couple comments.

**Comment by kidsonfilms-python-rules** [comment]
@rbrott should all fixed now!

**Pull request merged**